operator.go

```diff
@@ -741,12 +741,12 @@ func (o *operator) Run(ctx context.Context) error {
 		o.sw.Disable()
 	}
 	defer o.sw.Start().Stop()
-	o.capturers.captureStart(o.trails(), o.bookPath, o.desc)
-	defer o.capturers.captureEnd(o.trails(), o.bookPath, o.desc)
-	defer o.Close(true)
 	defer func() {
 		o.capturers.captureResult(o.trails(), o.Result())
+		o.capturers.captureEnd(o.trails(), o.bookPath, o.desc)
+		o.Close(true)
```

Comment:
Is it because not including `o.sw.Start().Stop()` in an anonymous function changes the behavior? This question comes from my lack of understanding of the defer specification.

Reply:
If we were to include it, the following

```go
o.sw.Start()
defer func() {
    o.sw.Stop()
}
```

Since I want to place Start() and Stop() in close proximity, I dare not include them in defer.